release/src/main/python-release/python_release_automation_utils.sh

```diff
@@ -132,6 +153,31 @@ function get_asc_name() {
 }
 
 
+#######################################
+# Create a new virtualenv and install the SDK
+# Globals:
+#   BEAM_PYTHON_SDK
+# Arguments:
+#   $1 - SDK type: tar, wheel
+#   $2 - python interpreter version: [python2.7, python3.5, ...]
+#######################################
+function install_sdk() {
+  sdk_file=$(get_sdk_name $1)
+  print_separator "Creating new virtualenv with $1 interpreter and installing the SDK from $sdk_file."
+  echo "test place 2"
```

Comment:
Left over code?

Reply:
That's a looks like a copy-pasted left-over code from previous revision. Do you want me to clean it up in this or a follow-up PR?

Reply:
Please clean up here, while you are at it.

Reply:
done